haswell/haswell_memory_bound.xml

```diff
@@ -109,22 +175,29 @@
             divideBySampleTime="false" />
         <display>
             <displayName>Stalled for other reasons</displayName>
-            <description>Fraction of cycles that are stalled for non-memory related reasons</description>
+            <description>Fraction of stalled cycles that are stalled for non-memory related reasons</description>
             <type>other</type>
             <colour>SpecialLine6</colour>
         </display>
     </metric>
+-->
 
     <metricGroup id="Haswell_papi_memory_boundedness">
         <displayName>MemoryBound</displayName>
         <description>Gives a measure of how memory bound an application is. This is only accurate on Intel Haswell (Xeon v3) cores</description>
         <metric ref="haswell.papi.active_cycles"/>
         <metric ref="haswell.papi.productive_cycles"/>
-        <metric ref="haswell.papi.stall_cycles" />
+        <metric ref="haswell.papi.stall_cycles"/>
+	<metric ref="haswell.papi.store_buffer_stall_cycles"/>
```

Comment:
Not sure if it's just github, but these don't look aligned.

Reply:
Indeed, there are tabs instead of spaces in the xml. I will fix it.